src/perl/vdotest/VDOTest.pm

```diff
@@ -489,7 +491,7 @@ sub getTGZNameForVersion {
 sub listSharedFiles {
   my ($self) = assertNumArgs(1, @_);
   my @files = ($self->SUPER::listSharedFiles(), @SHARED_FILES);
-  if ($self->{useUpstreamKernel}) {
+  if ($self->{useDmLinixModule}) {
```

Comment:
This is a typo: the new name is useDmLinuxModule.  You can test this by running the dmtest perl tests; they only take a minute or two right now and they use this option so you can know it's working.